app/src/main/java/com/framgia/wsm/screen/login/LoginModule.java

```diff
@@ -1,10 +1,18 @@
 package com.framgia.wsm.screen.login;
 
 import android.app.Activity;
+import android.content.Context;
 import android.support.annotation.NonNull;
 import com.framgia.wsm.data.source.UserRepository;
+import com.framgia.wsm.data.source.local.UserLocalDataSource;
+import com.framgia.wsm.data.source.local.sharedprf.SharedPrefsApi;
```

Comment:
ERROR: Unused import: com.framgia.wsm.data.source.local.sharedprf.SharedPrefsApi. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports